tests/e2e/healthcheck_test.go

```
@@ -0,0 +1,68 @@
+	t.Logf("E2E Redis healthy at %s", rawURL)
+}
+
+// TestHealthCheck_ServerHTTP verifies the E2E server responds to /healthz.
+func TestHealthCheck_ServerHTTP(t *testing.T) {
+	baseURL := e2eServerURL(t)
+	client := &http.Client{Timeout: 3 * time.Second}
+	resp, err := client.Get(baseURL + "/healthz")
+	if err != nil {
+		t.Fatalf("E2E server not reachable at %s/healthz: %v", baseURL, err)
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)